collect_app/src/main/res/values-am/strings.xml

```diff
@@ -11,4 +11,17 @@
   <!--%1$s represents degrees, %2$s represents minutes and %3$s represents seconds as in E 18°48'8"-->
   <!--%1$s represents degrees, %2$s represents minutes and %3$s represents seconds as in S 18°48'8"-->
   <!--%1$s represents degrees, %2$s represents minutes and %3$s represents seconds as in N 18°48'8"-->
+  <string name="ethiopian_month_meskerem">መስከረም</string>
```

Comment:
We'll need to update these in Transifex once the English file updates there. These files are all replaced by the latest from Transifex on release.